tsconfig.json

```diff
@@ -0,0 +1,28 @@
+{
+    "compilerOptions": {
```

Comment:
it would be awesome if you can write 1 line description for each of these that we can see what kind of configuration do we have in a glance.